src/table/toolbar/table-toolbar.component.ts

```diff
@@ -63,8 +63,12 @@ import { I18n, Overridable } from "../../i18n/i18n.module";
 	`
 })
 export class TableToolbar {
-	@Input() model: TableModel;
+@Input() model: TableModel;
 
+	@Input() set batchText (value: { BATCH_TEXT: string }) {
+		console.log(value.BATCH_TEXT);
+		this._batchText.override(value.BATCH_TEXT);
+	}
```

Comment:
Since this is just setting one value it should take a `value: string` and just apply that as the override ... I'm not sure what's going on with the `ariaLabel` taking an object ... that shouldn't be like that, but I'm assuming it's from before so there's not a lot of room to change it